build.gradle.kts

```
@@ -0,0 +1,217 @@
+}
+
+
+tasks.wrapper {
+  gradleVersion = "6.9"
+}
+
+tasks.jar.configure {
+  manifest {
+    attributes(
+      mapOf(
+        "Implementation-Title" to project.name,
+        "Implementation-Version" to version,
+        "Built-By" to System.getProperty("user.name"),
+        "Built-Date" to Date(),
+        "Built-JDK" to System.getProperty("java.version"),
+        "Built-Gradle" to gradle.gradleVersion
+      )
+    )
+  }
+}
+
+/* TESTING */
+tasks.test.configure {
+  testLogging {
+    showStandardStreams = true
+    exceptionFormat = FULL
+  }
+}
+
+sourceSets {
+  create("integTest") {
+    compileClasspath += main.get().output
+    runtimeClasspath += main.get().output
+  }
+}
+
+configurations {
+  named("integTestCompile").get().extendsFrom(testCompileClasspath.get())
+  named("integTestRuntime").get().extendsFrom(testRuntimeClasspath.get())
+}
+
+tasks.register("integTest", Test::class.java) {
+  testClassesDirs = sourceSets.getByName("integTest").output.classesDirs
+  classpath = sourceSets.getByName("integTest").runtimeClasspath
+  outputs.upToDateWhen { false }
+}
+/* TESTING */
+
+
+/* RELEASING */
+tasks.register<Jar>("sourceJar") {
+  from(sourceSets.main.get().allJava)
+  classifier = "sources"
+}
+
+tasks.register("javadocJar", Jar::class.java) {
+  dependsOn(tasks.javadoc)
+  from(tasks.javadoc.map { it.destinationDir!! })
+  classifier = "javadoc"
+}
+
+project.afterEvaluate {
+  tasks.register("writePom") {
+    val outputFile = file("$buildDir/pom/${project.name}-${project.version}.pom")
+    outputs.file(outputFile)
+
+    doLast {
+      maven {
+        pom {
+          project {
+            withGroovyBuilder {
+              "name"("App Engine Gradle Plugin")
+              "description"("This Gradle plugin provides tasks to build and deploy Google App Engine applications.")
+
+              "url"("https://github.com/GoogleCloudPlatform/app-gradle-plugin")
+              "inceptionYear"("2016")
+
+              "scm" {
+                "url"("https://github.com/GoogleCloudPlatform/app-gradle-plugin")
+                "connection"("scm:https://github.com/GoogleCloudPlatform/app-gradle-plugin.git")
+                "developerConnection"("scm:git://github.com/GoogleCloudPlatform/app-gradle-plugin.git")
+              }
+
+              "licenses" {
+                "license" {
+                  "name"("The Apache Software License, Version 2.0")
+                  "url"("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                  "distribution"("repo")
+                }
+              }
+              "developers" {
+                "developer" {
+                  "id"("loosebazooka")
+                  "name"("Appu Goundan")
+                  "email"("appu@google.com")
+                }
+              }
+            }
+          }
+        }.writeTo(outputFile)
+      }
+    }
+  }
+}
+
+
+// for kokoro releases
+tasks.register("prepareRelease", Copy::class.java) {
```

```suggestion
tasks.register<Sync>("prepareRelease") {
```

Most likely, "sync" task was assumed here, so the outcome does not carry stale files produced by copy tasks from the past executions.